agile-development/scrum.md

```diff
@@ -5,12 +5,12 @@
 Example: 60 - 120 minutes
 
 ### Prerequisites
-HTML  
-CSS  
-Git  
-GitHub  
-Editor  
-Desire to learn  
+- [HTML](https://github.com/Techtonica/curriculum/blob/master/html/html.md)  
```

Comment:
Please use relative paths. Example, replace:
`https://github.com/Techtonica/curriculum/blob/master/html/html.md`
With:
`../html/html.md`

Reply:
Do I need to change every path to a relative path?